### ISSUE TITLE
Push *all* options to io.connect()

### ISSUE THREAD

**Issue by psi-4ward** [balderdashy/sails.io.js#35]
Connecting the socket in [L 478](https://github.com/balderdashy/sails.io.js/blob/master/sails.io.js#L478) `self._raw = io(self.url, self);` passes `self` as opts object to socket.io but `SailsSocket(opts)` constructor in [L 314](https://github.com/balderdashy/sails.io.js/blob/master/sails.io.js#L314) *absorbs* only a few possible options:

```javascript
      // Absorb opts
      self.useCORSRouteToGetCookie = opts.useCORSRouteToGetCookie;
      self.url = opts.url;
      self.multiplex = opts.multiplex;
``` 

as quick fix i added  `_.(self, opts);`

**Comment by JetFault** [comment]
+1

**Comment by moehlone** [comment]
+1

**Comment by JetFault** [comment]
Bump on this

**Comment by sailsbot** [comment]
Thanks for posting, @psi-4ward.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
